tsi_core/README.rst

```diff
@@ -14,8 +14,8 @@ Building and Running
 This application can be built as follows:
 .. zephyr-app-commands::
    :zephyr-app:
-   :host-os: unix
-   :board: ek_tsi_skyp/m85 
+   :host-os:unix
+   :board:tsi/skyp/m85
```

Comment:
We should include tsi/skylp/m85 as well.

Reply:
In progress